scripts/verify-markdown.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright (C) 2020  Kevin Croft <krcroft@gmail.com>
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Based on `verify-bash.sh` by Patryk Obara <patryk.obara@gmail.com>
+# Copyright (C) 2019 licensed GPL-2.0-or-later
```

Comment:
Eh, you can remove me - additional copyright line won't cause problems for scripts updating years in the future :).

Reply:
sounds good; yeah, was just trying to show attribution - but yeah, too much of that would get cluttered!

Reply:
Done.